File: Java/Linear Search.java
public class Main {

	public static void main(String[] args) {
		int[]arr= {11,20,25,30,35,45,90};
		int target=25;
		System.out.println(linearSearch(arr,target));
   
	}
	static int linearSearch(int[]arr,int target) {
		if(arr.length==0) {
			return -1;
		}
		for(int index=0;index<arr.length;index++) {
			int element=arr[index];
			if(element==target) {
				return index;
			}
			
		}
		return -1;

	}

}

File: Add Code Here/C++/Searching/linearsearch.cpp
#include <iostream>

// Function to perform a linear search
int linearSearch(int arr[], int n, int target) {
    for (int i = 0; i < n; i++) {
        if (arr[i] == target) {
            return i; // Return the index of the target element if found
        }
    }
    return -1; // Return -1 if the target element is not found in the array
}

int main() {
    int arr[] = {10, 20, 30, 40, 50};
    int n = sizeof(arr) / sizeof(arr[0]);
    int target = 30;

    int result = linearSearch(arr, n, target);

    if (result != -1) {
        std::cout << "
